docs/auth/cloudflare/access.md

```diff
@@ -25,6 +25,8 @@ auth:
   providers:
     cfaccess:
       teamName: <Team Name>
+      serviceTokens:
+        "1uh2fh19efvfh129f1f919u21f2f19jf2.access": "bot-user@your-company.com"
```

Comment:
prettier is complaining; you may want to do `yarn prettier --write docs/auth/cloudflare/access.md`. I bet it wants to do something with these quotes